rulebook.tex

```diff
@@ -380,193 +435,196 @@ \subsection{Executive Committee 2015--2019}
 members of the Technical Committee. Executive Committee members are
 elected by the Board of Trustees and appointed by the President of the
 RoboCup Federation; they serve 3-year terms.\\[.5em]
-\emph{Ulrich Karras}, priv. Dozent, Essen, Germany\\
-\emph{Tim Niemueller}, RWTH Aachen University, Aachen, Germany
+\emph{Alexander Ferrein}, FH Aachen, Aachen, Germany\\
+\emph{Christian Deppe}, Festo Didactic SE, Denkendorf, Germany
 
 \subsection{Website and Mailinglist}
 \label{sec:website-ml}
 The website of the RoboCup Logistics League is available at\\
-\centerline{\url{http://www.robocup-logistics.org}}
+\centerline{\url{https://ll.robocup.org/}}
 
 \smallskip
 \noindent
 A mailing list for general announcements and discussion about the
 league is available at\\
 \centerline{\url{https://lists.kbsg.rwth-aachen.de/listinfo/robocup-logistics}}
 
-
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Competition Area} \label{sec:area}
```

Comment:
On the images of both,  if you still have acces to them and can modify them, I would move the red coordinate system a little bit, because right now it is hard to tell if there is a wall or not. Like below, I guess there is a wall, but it is hard to tell.

![image](https://user-images.githubusercontent.com/5959988/154744635-547473a3-c206-4bdf-812e-079c1367958f.png)

Reply:
Good point, fixed it